static/js-src/feature-page.js

```diff
@@ -32,6 +35,14 @@ const shareFeature = () => {
   }
 };
 
+function copyURLToClipboard() {
+  event.preventDefault();
+  const url = copyLinkEl.href;
+  navigator.clipboard.writeText(url).then(() => {
```

Comment:
Do we need to handle non-supporting browser? (like when this function is not available, or failed, show a failed copy toast, or fallback to the selection + execute('copy') method)

Reply:
No, I think our core users are all on modern browsers, which have supported this for many releases already.  For non-supporting browsers, the user has the work-around of simply copying the current browser URL the old fashioned way.